x/dex/client/cli/tx/tx_register_contract.go

```diff
@@ -16,10 +16,10 @@ var _ = strconv.Itoa(0)
 
 func CmdRegisterContract() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "register-contract [contract address] [code id] [need hook] [need order matching] [deposit] [dependency1,dependency2,...]",
+		Use:   "register-contract [contract address] [code id] [(deprecated)] [need order matching] [deposit] [dependency1,dependency2,...]",
```

Comment:
think we can just remove this flag here as only the spot-lite and vortex contract deployment scripts are using this cmd

Reply:
i think some infra tooling (e.g. loadtest) also relies on this so want to change it after we are done with all loadtesting